match.py

```diff
@@ -0,0 +1,176 @@
+"""Switch match."""
+
+import ipaddress
+
+from pyof.v0x01.common.flow_match import FlowWildCards
+
+IPV4_ETH_TYPE = 2048
+
+
+def format_request(request):
+    """Format user request to match function format."""
+    args = {}
+    for key, value in request.items():
+        args[key] = value
+    return args
+
+
+def do_match(flow_to_install, version, stored_flow_dict):
+    """Match a packet against this flow."""
+    if version == 0x01:
+        return match10(stored_flow_dict, flow_to_install)
+    elif version == 0x04:
+        return match13_no_strict(flow_to_install, stored_flow_dict)
```

Comment:
Do the two functions really need to have the parameters in the inverse order?

Reply:
Changed on commits https://github.com/kytos/flow_manager/pull/117/commits/4b1644d53e7617efd6ba030ffbdbc33671156e86, https://github.com/kytos/flow_manager/pull/117/commits/6defa10334890d9b2e9281e51a8d3432f6ccf9c3